libs/pmp-web/user/single-user-statistics/feature/src/lib/components/single-user-statistics-table/single-user-statistics-table.component.ts

```diff
@@ -1,20 +1,18 @@
-import { Component, Input, OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
 import { PrStatistics } from '@pimp-my-pr/shared/domain';
 
 @Component({
   selector: 'pmp-single-user-statistics-table',
   templateUrl: './single-user-statistics-table.component.html',
   styleUrls: ['./single-user-statistics-table.component.scss']
 })
-export class SingleUserStatisticsTableComponent implements OnInit {
+export class SingleUserStatisticsTableComponent {
   displayedColumns = ['id', 'title', 'waitingTime', 'linesToCheck', 'author', 'comments', 'link'];
 
   @Input()
   tableData: PrStatistics[];
 
-  constructor() {}
-
-  ngOnInit(): void {}
-
-  onNavigateToItem(any): void {}
+  onNavigateToItem(url: string): void {
+    window.open(url, '_blank');
```

Comment:
It would be better for type checking if you receive here a single PrStatistics object as an argument.